packages/teleterm/src/ui/DocumentCluster/ClusterResources/useServerSideResources.ts

```diff
@@ -30,14 +31,15 @@ export function addAgentLabelToQuery(filter: AgentFilter, label: AgentLabel) {
 const limit = 15;
 
 export function useServerSideResources<Agent>(
+  resourceKind: ResourceKind,
```

Comment:
What would you say to have here `defaultSort: SortType` instead of `resourceKind`?
We wanted the _fetch function_ to be passed as a parameter, it makes sense to treat _sorting_ the same way.